traffic_metadata.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package SMPS;
+
+option go_package = "spec;spec";
+
+message TrafficMetadata {
+  string load_balance_name = 1;
+  string load_balance_ip = 2;
+  string calico_hostendpoint = 3;
```

Comment:
I wonder if this should be generalized to just "host_endpoint".

Reply:
https://docs.projectcalico.org/introduction/ Calico is assigned a specific IP for traffic & security propose which is generalised by IBM & Red-Hat. I can add another field for host_endpoint if you say?

Reply:
@leecalcote fixed it!